assets/text/effects/arriveAtSite/languageLessons.properties

```diff
@@ -1,31 +1,32 @@
-~01~prompt~panel_013.(subject_is_abandonedDrauven)~1_narration=<subject> - I mean, [b][i]I[][] stayed until I was strong enough to move on and find home again.
-~01~prompt~panel_013.(subject_is_abandonedDrauven)~2_npc=?!
-~01~prompt~panel_014.(subject_is_abandonedDrauven)~1_narration=But while I was there I learned some of my first human words.
-~01~prompt~panel_014.(subject_is_abandonedDrauven)~2_npc=@*!!
-~01~prompt~panel_015.(subject_NOT_abandonedDrauven)~1_narration=Many years ago, I <subject.fluentInYandric:was/be> caught up in some... un-nice things.
-~01~prompt~panel_016.(subject_NOT_abandonedDrauven)~1_narration=I was part of a small warband. <npc2> would lead us to small human outposts.
-~01~prompt~panel_017.(subject_NOT_abandonedDrauven)~1_narration=We were pretty good at catching them and bringing them back to work in the fields and mines.
-~01~prompt~panel_018.(subject_NOT_abandonedDrauven)~1_narration=One day I realised the sounds it made were actually words, I was very surprised.
-~01~prompt~panel_019.(subject_NOT_abandonedDrauven)~1_narration=<npc2> already knew, even knew some of their words. <npc2.mf:He/She/They> told me to not to think about it...
-~01~prompt~panel_019.(subject_NOT_abandonedDrauven)~2_narration=... But I could not help wonder about if our roles were  reversed.
-~01~prompt~panel_019.(subject_NOT_abandonedDrauven)~2_narration~TWEAK={location:bottomRight}
-~01~prompt~panel_020.(subject_NOT_abandonedDrauven)~1_narration=So one day I weakened the human cages when nobody was looking.
-~01~prompt~panel_021~1_subject=<subject.abandonedDrauven:And in later years, sometimes I would still go back to look. Listen.<bookish.drauvenHealer:[face:sad]I think I was hoping to find you again./>/[face:grim]I think <npc2> suspected me - <npc2.mf:he/she/they> never seemed to trust me again after.\n... I hope you <subject.fluentInYandric:don't/not> think less of me because of what I used to do.>
-~01~prompt~panel_021~2_bookish=<subject.abandonedDrauven:/[face:talking]<bookish.friendLevel.subject:You did the best you could in a bad situation, <subject>. /I... Thank-you for telling me. You're one of us now, and I trust you./No, I think you did the right thing there./I think you've more than proven your good intentions, <subject>/Don't worry, I still trust you to have my back if things get rough!/[face:joy]Honestly, that sounds exactly like you, and I admire it!>>
-~01~prompt~panel_021~2_bookish~TWEAK={location:bottomRight,padXFraction:0.216,padYFraction:0.02,widthFraction:0.5402733}
-~01~prompt~panel_022~1_bookish=<subject.abandonedDrauven:And now you're here!/Anyhow,> <hero.exists:<hero> thinks/I think> we'll probably catch up to those <foes.plural> tomorrow, so don't forget to rest later.
+~01~prompt~panel_003~1_narration=But tonight looks like it is going to be another quiet night. Not that <subject> minds.
+~01~prompt~panel_004~1_speech=Ah, so that sound like that. <subject> gets it now.
```

Comment:
Do you think this should be wrapped in the <subject.fluentInYandric split to give better grammar if they're already proficient?